core/prisma/seed/seedCommunity.ts

```diff
@@ -824,7 +827,24 @@ export async function seedCommunity<
 		});
 
 	const createdMembers = possibleMembers?.length
-		? await trx.insertInto("members").values(possibleMembers).returningAll().execute()
+		? await trx
+				.with("community_membership", (db) =>
+					db.insertInto("community_memberships").values(
+						possibleMembers.map((m) => ({
+							...m,
+							id: m.id as unknown as CommunityMembershipsId,
```

Comment:
Here, and everywhere else that I've modified the membership write queries, I am ensuring that the IDs will be the same between the tables, which will make inserting data from `members` into `community_memberships` easier. In other places I just select the id from the first CTE to use in the later query, but since this code will all be deleted soon I took the easy route of generating the ids on the server and assuming there won't be a collision

Reply:
nice solution to the problem of keeping the value of the `MemberSelect` field accurate!